doc/source/examples/constructing_universe.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
```

Comment:
<p>The outputs here are going to change whenever we update the mda / nglview versions, is this going to mess with CI?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>There's a sanitization regex rule on <code>sanitize_output.cfg</code> for these messages:</p><p><code></code><code></p><p>[Version] </p><p>regex: (Using .* version.*) </p><p>replace: VERSION</p><p> </code><code></code></p><p> </p>